2-library/domain-model-6core-java/src/main/java/com/_6core/lib/java/domain/entities/v1/warehouse/Warehouse.java

```diff
@@ -0,0 +1,13 @@
+package com._6core.lib.java.domain.entities.v1.warehouse;
```

Comment:
1. dont use plural form for packages. All our packages named in singular already. Dont violate this convention.
2. we need IMPLEMENT EXISTING interfaces, nor create a new one. 
```suggestion
package com._6core.lib.java.domain.``.v1.warehouse;
```